packages/ui/primitives/avatar.tsx

```diff
@@ -72,7 +74,9 @@ const AvatarWithText = ({
       <AvatarFallback className="text-xs text-gray-400">{avatarFallback}</AvatarFallback>
     </Avatar>
 
-    <div className="flex flex-col text-left text-sm font-normal">
+    <div
+      className={cn('hidden flex-col text-left text-sm font-normal lg:flex', showText && 'flex')}
+    >
```

Comment:
This doesn't make sense

If I were to set `showText` as false, I'd expect it to not be shown for all screen sizes.

Maybe pass in a `textSectionClassName` instead for this?